src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -137,87 +137,94 @@ public override MetadataFile FindMetadataFile(Series series, string path)
+                    tvShow.Add(uniqueId);
 
-                    foreach (var tag in tags)
+                    if (series.ImdbId.IsNotNullOrWhiteSpace())
                     {
-                        tvShow.Add(new XElement("tag", tag.Label));
+                        var imdbId = new XElement("uniqueid", series.ImdbId);
+                        imdbId.SetAttributeValue("type", "imdb");
+                        tvShow.Add(imdbId);
                     }
-                }
 
-                if (series.FirstAired.HasValue)
-                {
-                    tvShow.Add(new XElement("premiered", series.FirstAired.Value.ToString("yyyy-MM-dd")));
-                }
+                    foreach (var genre in series.Genres)
+                    {
+                        tvShow.Add(new XElement("genre", genre));
+                    }
 
-                tvShow.Add(new XElement("studio", series.Network));
+                    if (series.Tags.Any())
+                    {
+                        var tags = _tagService.GetTags(series.Tags);
 
-                foreach (var actor in series.Actors)
-                {
-                    var xmlActor = new XElement("actor",
-                        new XElement("name", actor.Name),
-                        new XElement("role", actor.Character));
+                        foreach (var tag in tags)
+                        {
+                            tvShow.Add(new XElement("tag", tag.Label));
+                        }
+                    }
 
-                    if (actor.Images.Any())
+                    if (series.FirstAired.HasValue)
                     {
-                        xmlActor.Add(new XElement("thumb", actor.Images.First().Url));
+                        tvShow.Add(new XElement("premiered", series.FirstAired.Value.ToString("yyyy-MM-dd")));
                     }
 
-                    tvShow.Add(xmlActor);
-                }
+                    tvShow.Add(new XElement("studio", series.Network));
 
-                var doc = new XDocument(tvShow);
-                doc.Save(xw);
+                    foreach (var actor in series.Actors)
+                    {
+                        var xmlActor = new XElement("actor",
+                            new XElement("name", actor.Name),
+                            new XElement("role", actor.Character));
 
-                _logger.Debug("Saving tvshow.nfo for {0}", series.Title);
+                        if (actor.Images.Any())
+                        {
+                            xmlActor.Add(new XElement("thumb", actor.Images.First().Url));
+                        }
 
-                return new MetadataFileResult("tvshow.nfo", doc.ToString());
+                        tvShow.Add(xmlActor);
+                    }
+
+                    var doc = new XDocument(tvShow);
+                    doc.Save(xw);
+
+                    xmlResult += doc.ToString();
+                    xmlResult += Environment.NewLine;
```

Comment:
Instead of adding a new line at the end that may need to get trimmed off, add the new before the URL only when `Settings.SeriesMetadata` is enabled.